source/includes/steps-fle-convert-to-a-remote-master-key-gcp.yaml

```
@@ -1,50 +1,91 @@
-title: Create an AWS IAM User
-ref: create-an-aws-iam-user
+title: Create a GCP Service Account
+ref: create-a-gcp-service-account
 content: |
-  Create a new programmatic IAM user in the AWS management console.
-  CSFLE-enabled clients authenticate with AWS KMS using the IAM user to
-  encrypt and decrypt the remote master key. The IAM user must be granted
-  full ``List`` and ``Read`` permissions for the KMS service.
 
-  .. note:: Client IAM User Credentials
+  Create a service account for your client application by following the
+  official Google documentation on
+  `Creating and managing a service account <https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating>`__.
+
+  After you create the service account, locate and record the following
+  authentication settings for use in a later step:
+
+  - **email**
+  - **privateKey** (this value is only returned when you create the account)
+  - **endpoint**
 
-     The CSFLE-enabled client uses the IAM User's :guilabel:`Access Key
-     ID` and :guilabel:`Secret Access Key` as configuration values. Take
-     note of these and reference them when we update the client.
 ---
 title: Create the Master Key
-ref: create-the-master-key
+ref: create-the-master-key-gcp
 content: |
 
   The following diagram shows how the **master key** is created and stored
-  when using a KMS provider:
+  on a KMS provider:
 
   .. image:: /figures/CSFLE_Master_Key_KMS.png
      :alt: Diagram that describes creating a master key when using a KMS provider
 
-  In AWS management console, create a new symmetric master key in the KMS
-  section. Choose a name and description that helps you identify it; these
-  fields do not affect the functionality or configuration.
+  Create a **symmetric key** using the official Google Cloud
+  `Creating symmetric keys guide <https://cloud.google.com/kms/docs/creating-keys>`__.
+  This is your master key which you can access from your application.
+
+  .. note::
+
+     If you rotate the key whether by specifying a date or manually, the
+     provider changes the content of the master key. In order to transition
+     to using the new key, you need to perform the following steps:
```

Suggestion: Conciseness. Double-check me here to make sure I haven't lost any meaning, but I don't think I have.

 ```suggestion
     provider changes the content of the master key. To transition
     to the new key:
```